tensorflow/lite/examples/label_image/label_image.cc

```diff
@@ -26,19 +28,21 @@ limitations under the License.
 #include <fstream>
 #include <iomanip>
 #include <iostream>
+#include <map>
 #include <memory>
 #include <sstream>
 #include <string>
 #include <unordered_set>
 #include <vector>
 
-#include "tensorflow/lite/examples/label_image/bitmap_helpers.h"
-#include "tensorflow/lite/examples/label_image/get_top_n.h"
+#include "tensorflow/lite/delegates/nnapi/nnapi_delegate.h"
 #include "tensorflow/lite/kernels/register.h"
-#include "tensorflow/lite/model.h"
 #include "tensorflow/lite/optional_debug_tools.h"
 #include "tensorflow/lite/profiling/profiler.h"
 #include "tensorflow/lite/string_util.h"
+#include "tensorflow/lite/tools/evaluation/utils.h"
+#include "tensorflow/lite/examples/label_image/bitmap_helpers.h"
```

Comment:
`examples/` after `delegates/` & before `kernels/`

Reply:
These two headers are in label_image. I think they are "7. Your project's .h files" described the [coding style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)". Do I read it right?